docker/README.md

```diff
@@ -0,0 +1,25 @@
+# debos-docker
+
+Docker container for ['debos' tool](https://github.com/go-debos/debos).
+
+## Installation
+```
+docker pull debos/debos-docker
+```
+
+Debos needs virtualization to be enabled on the host and shared with the container.
+
+Check that `kvm` is enabled by running ```ls /dev/kvm```
+
+## Usage
+To build `recipe.yaml`:
+```
+cd <PATH_TO_RECIPE_DIR>
+docker run --rm --device /dev/kvm --group-add $(getent group kvm | cut -d: -f3) -w /recipes -u $(id -u) -i -v $(pwd):/recipes -t debos/debos-docker debos <RECIPE.yaml>
```

Comment:
kvm group is specific o some operating systems; I'd overall recomend making `/dev/kvm` world writable or add it to uaccess (which is the default anyway on most systems);  So the `--group-add` should probably be dropped and replaced by a note in the kvm setup that it should be ensured to be writable by the user running in the docker container.